staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -304,6 +304,9 @@ type KubeletConfiguration struct {
 	// Default: "4h"
 	// +optional
 	StreamingConnectionIdleTimeout metav1.Duration `json:"streamingConnectionIdleTimeout,omitempty"`
+	// cloudProvider is the provider for cloud services.
```

Comment:
copy the flag documentation here (meaning of empty string, etc), and maybe also call out the special "external" value?

Reply:
We should include the same details as the flag help, e.g.:
"Specify empty string for running with no cloud provider. If set, the cloud provider determines the name of the node (consult cloud provider documentation to determine if and how the hostname is used)."